calculatorTest/GeneralSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CalculatorTest
+{
+    public static class GeneralSettings
+    {
+        public const string BASEURL = "http://localhost:64177/";
+
+    }
```

Comment:
Please remove empty line.

```suggestion
        public const string BASEURL = "http://localhost:64177/";
    }
```